x-pack/test/reporting/api/generation_urls.js

```diff
@@ -7,6 +7,11 @@
 // These all have the domain name portion stripped out. The api infrastructure assumes it when we post to it anyhow.
 
 /* eslint-disable max-len */
+
+/*The URL below was captured from release 6.4 and then the layout section was removed to test structure before
```

Comment:
use `//` instead of `/*` for the comment.  (above thing has to be `/*` since it's a special comment for the linter)